main.go

```diff
@@ -0,0 +1,334 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/andygrunwald/go-jira"
+	"github.com/go-redis/redis"
+	"github.com/mmcdole/gofeed"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"jaytaylor.com/html2text"
+	"log"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+	"time"
+)
+
+var addr = flag.String("listen-address", ":8080", "The address to listen on for HTTP requests.")
+var lastRunGauge prometheus.Gauge
+var issuesCreatedCounter prometheus.Counter
+var issueCreationErrorCounter prometheus.Counter
```

Comment:
Style nitpick: these could be grouped into a single var declaration.
```suggestion
var (
    lastRunGauge prometheus.Gauge
    issuesCreatedCounter prometheus.Counter
    issueCreationErrorCounter prometheus.Counter
)
```